### PR TITLE
[gl] Galician Locale Translation

### PR DESCRIPTION
**Pull request by nunhes**
**Description**
Galician Locale Translation

**Checklist:**
- [x] Translate to Galician
- [x] Create GalicianTranslationTest
- [ ] Test Passed
- [x] Include changes on README.md. Galician language added

### CODE REVIEW

**Comment by kenjis** [comment]
Thank you for sending this PR!

You must GPG-sign your work, certifying that you either wrote the work or otherwise have the right to pass it on to an open-source project. See Developer's Certificate of Origin. See [signing][1].
**Note that all your commits must be signed.** If you have an unsigned commit, you can sign the previous commits by referring to [gpg-signing-old-commits][2].

[1]: https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing
[2]: https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#gpg-signing-old-commits

File: tests/Language/AbstractTranslationTestCase.php [hunk 1]
```diff
@@ -47,6 +47,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         GreekTranslationTest::class              => 'el',
         SpanishTranslationTest::class            => 'es',
         FarsiTranslationTest::class              => 'fa',
+        GalicianTranslationTest::class           => 'gl',
         FrenchTranslationTest::class             => 'fr',
```
Comment (kenjis) [inline tests/Language/AbstractTranslationTestCase.php:51]: Please fix this error:
```diff
1) Translations\Tests\AutoReview\TranslationsCodeTest::testLocalesArrayAreArrangedByLocaleCode
The `AbstractTranslationTestCase::$locales` property should be arranged by locale code.
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
     'Translations\Tests\GreekTranslationTest' => 'el'
     'Translations\Tests\SpanishTranslationTest' => 'es'
     'Translations\Tests\FarsiTranslationTest' => 'fa'
+    'Translations\Tests\GalicianTranslationTest' => 'gl'
     'Translations\Tests\FrenchTranslationTest' => 'fr'
-    'Translations\Tests\GalicianTranslationTest' => 'gl'
     'Translations\Tests\HungarianTranslationTest' => 'hu'
     'Translations\Tests\IndonesianTranslationTest' => 'id'
     'Translations\Tests\ItalianTranslationTest' => 'it'

/home/runner/work/translations/translations/tests/AutoReview/TranslationsCodeTest.php:34
phpvfscomposer:///home/runner/work/translations/translations/vendor/phpunit/phpunit/phpunit:106

FAILURES!
Tests: 3, Assertions: 73, Failures: 1.
Error: Process completed with exit code 1.
```
https://github.com/codeigniter4/translations/actions/runs/7944239245/job/21691254003?pr=437

Comment (nunhes) [reply tests/Language/AbstractTranslationTestCase.php]: I repair this error

**Comment by kenjis** [comment]
Thank you for fixing `TranslationsCodeTest`.

The four commits are not signed yet. Sign all commits.
I recommend you set up *git* to sign all commits automatically, and squash this branch.
Screenshot 2024-02-19 13 39 12

**Comment by kenjis** [comment]
There are missing translations. "CLI.generator.className.test", "Validation.field_exists".
Also we cannot merge commits without signing.
If you can't sign old commits, please create a new PR.

1) Translations\Tests\GalicianTranslationTest::testAllConfiguredLanguageKeysAreIncluded with data set "gl" ('gl')
Failed asserting that the language keys "CLI.generator.className.test", "Validation.field_exists" in the main repository are included for translation in "gl" locale.
Failed asserting that an array is empty.

**Comment by kenjis** [comment]
"CLI.generator.className.test" and  "Validation.field_exists" are missing.

```
There was 1 failure:

1) Translations\Tests\GalicianTranslationTest::testAllConfiguredLanguageKeysAreIncluded with data set "gl" ('gl')
Failed asserting that the language keys "CLI.generator.className.test", "Validation.field_exists" in the main repository are included for translation in "gl" locale.
Failed asserting that an array is empty.

/home/runner/work/translations/translations/tests/Language/AbstractTranslationTestCase.php:170

FAILURES!
Tests: 7, Assertions: 7, Failures: 1.
All tested locales have failed!
```
https://github.com/codeigniter4/translations/actions/runs/8693596874/job/23841305344?pr=437

**Comment by paulbalandan** [comment]
Hi @nunhes , will you be able to address the test failure?

```
There was 1 failure:

1) Translations\Tests\GalicianTranslationTest::testAllConfiguredLanguageKeysAreIncluded with data set "gl" ('gl')
Failed asserting that the language keys "CLI.generator.className.test", "Validation.field_exists" in the main repository are included for translation in "gl" locale.
Failed asserting that an array is empty.

/home/runner/work/translations/translations/tests/Language/AbstractTranslationTestCase.php:170
```

**Comment by paulbalandan** [comment]
Closing for no activity for almost 2 months now.

**Pull request closed without merging**